torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -786,18 +787,19 @@ def create_wrapped_node(self, tx, args, kwargs):
         # See NOTE [HigherOrderOperator tracing design] for more details
         checkpoint = tx.copy_graphstate()
         graph_checkpoint = tx.output.graph
+        flattened_kwargs = [kwargs[key] for key in sorted(kwargs.keys())]
+
         (
             body_r,
             body_graph,
             body_lifted_freevars,
         ) = speculate_subgraph(
             tx,
-            args[0],
-            [
-                *args[1:],
-            ],
+            args[0],  # function
+            [*(args[1:] + flattened_kwargs)],
             graph_checkpoint,
             checkpoint,
+            add_arg_to_subgraph=False,
```

Comment:
`add_arg_to_subgraph=False` should probably be the default, we will want to move all the other HigherOrderOps to this

Reply:
Was planning to do this in follow-up PR as this would require few changes.

Reply:
sounds good